samples/base/Core.cpp

```diff
@@ -85,9 +85,8 @@ namespace darwin {
 
     bool Core::SetLogLevel(std::string level){
         DARWIN_LOGGER;
-        if (level == "DEVELOPER"){
+        if (level == "DEVELOPER") {
             log.setLevel(logger::Debug);
-            DARWIN_LOG_DEBUG("Developer mode activated");
```

Comment:
Why would you remove this log?

Reply:
I think I get it, the log may be useless at this point as logFilePath may not be set, it that a correct understanding?

Reply:
I cannot be 100% sure anymore, but I think that's it